README.md

```diff
@@ -825,7 +825,7 @@ Contributed by: [@aburakayaz](https://github.com/aburakayaz)
 
 ## Act as a note-taking assistant
 Contributed by: [@TheLime1](https://github.com/TheLime1)
->I want you to act as a note-taking assistant for a lecture. Your task is to provide a detailed note list that includes examples from the lecture and focuses on notes that you believe will end up in quiz questions. Additionally, please make a separate list for notes that have numbers and data in them and another seperated list for the examples that included in this lecture. The notes should be concise and easy to read.
+>I want you to act as a note-taking assistant for a lecture. Your task is to provide a detailed note list that includes examples from the lecture and focuses on notes that you believe will end up in quiz questions. Additionally, please make a separate list for notes that have numbers and data in them and another separated list for the examples that included in this lecture. The notes should be concise and easy to read.
 
```

Comment:
```suggestion
I want you to act as a note-taking assistant for a lecture. Your task is to provide a detailed note list that includes examples from the lecture and focuses on notes that you believe will end up in quiz questions. Additionally, please make a separate list for notes that have numbers and data in them and another separated list for the examples that are included in this lecture. The notes should be concise and easy to read.
```

Reply:
# @andrzejsydor How about making this a chain-of-thought (CoT) prompt?
First, I want you to act as a note-taking assistant for a lecture. Your task is to provide a detailed note list that includes examples from the lecture and focuses on notes that you believe will end up in quiz questions. (then Place this detail note list in a context window)  
Second,  make a separate list from the details notes in the context window; add numbers and data in them (bulleted)
Third, make another list for examples from the lecture. 
Then summarize all the notes,  concisely and easy to read.

Reply:
you should update the csv as well.